tests/diffpy/utils/scattering_objects/test_diffraction_objects.py

```diff
@@ -231,94 +231,179 @@ def test_diffraction_objects_equality(inputs1, inputs2, expected):
     assert (diffraction_object1 == diffraction_object2) == expected
 
 
-def test_q_to_tth():
-    # Valid q values that should result in 0-180 tth values after conversion
+def _test_valid_diffraction_objects(actual_diffraction_object, function, expected_array):
+    """Checks the behavior of the DiffractionObject:
+    when there is no wavelength, we expect the correct warning message and output,
+    otherwise, we only check the output matches the expected array."""
+    if actual_diffraction_object.wavelength is None:
+        with pytest.warns(UserWarning) as warn_record:
+            getattr(actual_diffraction_object, function)()
+        assert str(warn_record[0].message) == (
+            "INFO: no wavelength has been specified. You can continue "
+            "to use the DiffractionObject but some of its powerful features "
+            "will not be available. To specify a wavelength, you can use "
+            "DiffractionObject(wavelength=0.71)."
```

Comment:
0.71?  Or something like "wavelength=<wavelength>) where you replace <wavelength> with the actual wavelength in the units of..." Etc.   Also, are these the correct instructions? This seems to be instantiating a new DO not adding a wavelength to an existing DO.

Let's also make sure to add these instructions to the documentation.